libs/core/langchain_core/output_parsers/json.py

```diff
@@ -44,87 +45,6 @@ def _custom_parser(multiline_string: str) -> str:
     return multiline_string
 
 
-# Adapted from https://github.com/KillianLucas/open-interpreter/blob/5b6080fae1f8c68938a1e4fa8667e3744084ee21/interpreter/utils/parse_partial_json.py
-# MIT License
-def parse_partial_json(s: str, *, strict: bool = False) -> Any:
```

Comment:
ooc why move this? output_parsers/json still seems like a reasonable place for it, were there cyclic imports?

Reply:
This was to resolve cyclic imports, but since we are no longer parsing tool calls on __add__ (since we'd need to adjust across providers) I moved this back.